configs/webserver/sample.conf

```diff
@@ -95,8 +95,10 @@ max_file_upload_size = 4294967296
 #page = ""
 
 [pipeline]
+# Endpoint to the pipeline service
 #endpoint = "http://mlops.com:9500"
```

Comment:
It is be safe to use arbitrary URLs that may actually exist and get hooked by malicious entities.  Use `example.com`, `localhost`, or a URL with a non-existent TLDs.

Reply:
FYI: https://www.iana.org/help/example-domains